packages/react-core/src/components/Button/Button.tsx

```diff
@@ -52,6 +52,8 @@ export interface ButtonProps extends React.HTMLProps<HTMLButtonElement>, OUIAPro
   tabIndex?: number;
   /** Adds small styling to the button */
   isSmall?: boolean;
+  /** Adds large styling to the button */
+  isLarge?: boolean;
```

Comment:
In retrospect we probably should have combined the isLarge and isSmall.

Reply:
Maybe in the next breaking change release we can refactor them into a `size` property.